src/styles/main.scss

```diff
@@ -1,3 +1,64 @@
+@import "variables.scss";
+
+* {
+  background-color: #eee;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap};
+  padding: #{$padding};
+  width: $width * 7 + $gap * 6;
+
+  &__day {
+    height: #{$height};
+    width: #{$width};
+    border: 1px solid black;
+    box-sizing: border-box;
```

Comment:
this comment about  `box-sizing: border-box;` property, not about `background-color: #eee;`
now it looks suboptimal

switch mentioned properties places.
Some explanation: `box-sizing: border-box;` is the preferable value and if you work with such a setting it should be the same for all elements on the page to avoid confusing, so the `*` selector is correct place for it